azure-pipelines.yml

```diff
@@ -196,7 +196,7 @@ jobs:
   displayName: Build Homebrew Formula
 
   dependsOn: BuildPythonWheel
-  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'))
+  condition: succeeded()
```

Comment:
Before this PR is merged, I will revert this change so that homebrew builds are not done as part of PR validation, only CI.

Reply:
Updated in 2f9cc5a